### PR TITLE
chore: update Qt to 6.7.2

### PR DESCRIPTION
**Pull request by edgars-cirulis**

### CODE REVIEW

**Comment by edgars-cirulis** [comment]
Been using for a while on win10, also on win11 haven't found any issues. 
image

**Comment by edgars-cirulis** [comment]
Shall I close this?

**Review by 3gf8jv4dv — APPROVED** [review]
> Shall I close this?

I personally approve this PR. I suggest you wait for approval from at least two other maintainers.

**Review by TheKodeToad — APPROVED** [review]

**Review by DioEgizio — APPROVED** [review]

**Review by Trial97 — APPROVED** [review]

**Pull request merged**
